spec/lib/miq_automation_engine/miq_ae_service_model_spec.rb

```diff
@@ -80,4 +80,18 @@ module MiqAeServiceModelSpec
       end
     end
   end
+
+  describe MiqAeMethodService::MiqAeServiceMiqAeDomain do
+    let(:tenant) { Tenant.seed }
+    let(:domain) { FactoryGirl.create(:miq_ae_domain, :tenant => tenant) }
+
+    it "#ae_domains" do
+      domain
+      t = MiqAeMethodService::MiqAeServiceTenant.new(tenant)
+      dom = t.ae_domains.first
+      [:name, :system, :priority, :id].each do |attr|
```

Comment:
@mkanoor Don't need to change this but as of Ruby 2.0 you can use `%i(name system priority id)` to get an array of symbols.  Also, `%w(...)` would work here as well.